tools/batchTestGenerator/internal/config.go

```diff
@@ -56,8 +56,22 @@ type ClusterTarget struct {
 }
 
 type Target struct {
-	Name   string `json:"name"`
-	Region string `json:"region"`
+	Name            string   `json:"name"`
+	Region          string   `json:"region"`
+	ExcludedTests   []string `json:"excluded_tests"`
+	excludedTestMap map[string]struct{}
```

Comment:
```suggestion
	excludedTestSet map[string]struct{}
```

You're using it as a set, should probably name it that way.

Reply:
Done!